apps/verify/models/cookie.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const uuidv1 = require('uuid/v1');
+const read = require('./read-cookie');
+const write = require('./write-cookie');
+
+const readUuid = async(uuid) => {
+  return await read(uuid);
+};
+
+const writeCookie = async(data) => {
+  data.uuid = uuidv1();
```

Comment:
UUIDv1 is an interesting choice -- they're predictable, so if you know one you can predict another.
Consider using v4, which is more properly random.